packages/cli/bin/executable.js

```diff
@@ -0,0 +1,14 @@
+// This file is used as a entry point for executable with NODE_ENV=executable
+process.env.NODE_ENV = 'executable';
+// ensure that we're running within a supported node version
```

Comment:
Also include the asset discovery browser in executable - [ref](https://docs.percy.io/docs/caching-asset-discovery-browser-in-ci) make it default.

[optional] create separate variant of executable which doesn't includes browser, so user can specify browser themselves if they need instead of downloading bloated binary.

Reply:
the current executable doesn’t include pre-downloade browser, I tried creating one with installed browser by adding export PERCY_POSTINSTALL_BROWSER=true before yarn install step but it didn’t work for me. also if we want include browser we will have to create diff github actions for each platform right so that it downloads correct chromium browser?

Reply:
hmm, lets tackle this in followup.